shared/commonUtil.ts

```diff
@@ -75,3 +92,7 @@ export const faceMatchConfig = (resp: string) => {
     },
   };
 };
+
+export const getBackupFileName = () => {
```

Comment:
There is one more backup file name const defined in fileStorage. Let's have it single value only